File: C++Programming/Practice/program26.cpp
// Print 0 to N on screen

#include<iostream>
using namespace std;

void Display(int iNo)
{
    int iCnt = 0;

    // filter
    if(iNo < 0)
    {
        cout<<"Invalid input";
        return;
    }

    for(iCnt = 0; iCnt <= iNo; iCnt++)
    {
        cout<<iCnt<<"\n";
    }
}

int main()
{
    int iValue = 0;
    
    cout<<"Enter frequency :\n";
    cin>>iValue;

    Display(iValue);

    return 0;
}

File: Java_Programming/Practice/program26.java
// print "Jay Ganesh" as per user demand

import java.util.Scanner;

class Iteration
{
    public void Display(int iNo)
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.println("Jay Ganesh...");
        }
    }

}
class program26
{

    public static void main(String A[])
    {
        int iValue = 0;

        Scanner sobj = new Scanner(System.in);
        Iteration iobj = new Iteration();

        System.out.println("Enter value : ");
        iValue = sobj.nextInt();
        
        iobj.Display(iValue);
        
   
    }
}